scripts/process_vulnerabilities.py

```
@@ -264,21 +302,47 @@ def load_safety_report(report_path: Path) -> List[Dict]:
-            
-        # Safety CLI output format
-        vulnerabilities = data.get("vulnerabilities", [])
+
+        # Safety CLI output format - return vulnerabilities list
         if vulnerabilities:
-            print(f"✅ Successfully parsed {len(vulnerabilities)} vulnerabilities from report")
+            # Validate each vulnerability has required fields
+            valid_vulns = []
+            for vuln in vulnerabilities:
+                if not isinstance(vuln, dict):
+                    print(f"⚠️  Skipping invalid vulnerability entry (not a dict)")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)